pom.xml

```diff
@@ -416,8 +467,6 @@ since this is impossible for AWS S3 dependencies without Maven.
             <groupId>edu.ucar</groupId>
             <artifactId>netcdfAll</artifactId>
             <version>5.5.3</version>  <!-- version number duplicated several times below -->
-            <scope>system</scope>
-            <systemPath>${project.basedir}/WEB-INF/lib/netcdfAll-5.5.3.jar</systemPath>
```

Comment:
I haven't tested with the system scope and systemPath configs removed like this, but if it works for you, then :+1: for dropping them here.  

This results in the netcdfAll jar being packaged with the WAR file when built with the standard Maven build, right?  Is that desired?

Reply:
Yeah, in my testing the netcdf jar is included in the war without these lines. Worst case if it doesn't work reliably, we can re-add them.